metricflow/model/objects/elements/measure.py

```diff
@@ -20,6 +20,13 @@ class NonAdditiveDimensionParameters(HashableBaseModel):
     window_groupings: List[str] = []
 
 
+class MeasureAggregationParameters(HashableBaseModel):
+    """Describes parameters for aggregations"""
+
+    percentile: Optional[float] = None
+    disc: bool = False
```

Comment:
This parameter should be optional as well.

Reply:
or by default its false, we don't want it ever to be None, correct? so it should be set to false.